polyfill/lib/ecmascript.mjs

```diff
@@ -79,6 +81,7 @@ export const ES = ObjectAssign({}, ES2019, {
     if (!(id in BUILTIN_CALENDARS)) throw new RangeError(`unknown calendar ${id}`);
     return new BUILTIN_CALENDARS[id]();
   },
+  GetDefaultCalendar: () => ES.GetBuiltinCalendar('iso8601'),
```

Comment:
Where is `ES.GetBuiltinCalendar` implemented?  Is it already checked in?

Reply:
Yes, it was checked in with the Temporal.Calendar stub in da58923e5ab1ec8beec6116399aee5d22b8eeb16.